querybook/webapp/components/DataDocSchedule/DataDocScheduleForm.tsx

```diff
@@ -191,26 +202,7 @@ export const DataDocScheduleForm: React.FunctionComponent<
                             </b>
                             , and scheduled queries will also run by them.
                         </div>
```

Comment:
we need to update this help text accordingly. 

@czgu you mentioned last time that you'd like to have the schedule owner to be notified by default, could you elaborate?

Reply:
let's forget about the default notification for now and remove this help text